client/sttp-client/src/test/scalanative/sttp/tapir/client/sttp/SttpClientTests.scala

```diff
@@ -0,0 +1,58 @@
+package sttp.tapir.client.sttp
+
+import scala.util.Try
+import cats.effect.IO
+
+import sttp.tapir.{DecodeResult, Endpoint}
+import sttp.tapir.client.tests.ClientTests
+import sttp.client3._
+
+object Backend  {
+  val backend = CurlTryBackend(verbose = false)
```

Comment:
out of interest, any reason to use `CurlTryBackend` instead of `CurlBackend`?

Reply:
ah, probably easier to convert to IO?

Reply:
Not knowing the implementation of neither of them CurlTryBackend provides more type info in the signature, it looks purer.  Also easier to convert to IO without an IO.delay